lib/main.dart

```diff
@@ -36,29 +36,254 @@ class _MyHomePageState extends State<MyHomePage> {
+            padding: EdgeInsets.only(left: 8.0, right: 8.0),
+            child: Row(
+              children: [
+                Icon(Icons.stars_sharp, color: Color(0xFF008080)),
+                Text("4,8 (485)"),
+                Spacer(),
+                Text(
+                  "9,99 zł",
+                  style: TextStyle(fontSize: 24, color: Color(0xFF008080), fontWeight: FontWeight.bold),
+                ),
+              ],
+            ),
+          ),
+          const Padding(
+            padding: EdgeInsets.only(left: 8.0, right: 8.0),
+            child: Row(
+              children: [
+                Icon(Icons.timer, color: Color(0xFF008080)),
+                Text("Odbiór: 10:00 - 14:00"),
+              ],
+            ),
+          ),
+          const Divider(color: Colors.grey),
+          const Row(
+            children: [
+              Padding(
+                padding: EdgeInsets.all(8.0),
+                child: Text(
+                  "CO JEST W OFERCIE?",
+                  style: TextStyle(fontSize: 14, fontWeight: FontWeight.bold),
+                  textAlign: TextAlign.left,
+                ),
+              ),
+            ],
+          ),
+          const Padding(
+            padding: EdgeInsets.all(8.0),
+            child: SizedBox(
+              height: 35,
+              child: Text(
+                "To niespodzianka - Paczka będzie wypełniona pysznym jedzonkiem, które nie sprzedało się...",
+              ),
+            ),
+          ),
+          const Padding(
+            padding: EdgeInsets.only(left: 16.0, top: 8.0, bottom: 8.0),
+            child: Row(
+              children: [
+                Text(
+                  "Artykuły Spożywcze      Wegetariańskie",
+                  style: TextStyle(fontSize: 14, color: Color(0xFF008080), fontWeight: FontWeight.bold),
+                ),
+              ],
+            ),
+          ),
+          const Divider(color: Colors.grey),
+          const Column(
+            children: [
+              Padding(
+                padding: EdgeInsets.only(top: 8.0, bottom: 8.0),
+                child: Text("CO MÓWIĄ INNI", style: TextStyle(fontWeight: FontWeight.bold)),
+              ),
+              Padding(
+                padding: EdgeInsets.only(bottom: 16.0),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    Icon(Icons.stars_sharp, size: 40, color: Color(0xFF008080)),
+                    Text(
+                      "4,8/5,0",
+                      style: TextStyle(fontWeight: FontWeight.bold, fontSize: 32),
+                    )
+                  ],
+                ),
+              ),
+              Divider(color: Colors.grey),
+              Text("3 NAJLEPSZE RZECZY", style: TextStyle(fontWeight: FontWeight.bold, color: Color(0xFF008080))),
+              Padding(
+                padding: EdgeInsets.only(bottom: 8.0, top: 16.0, left: 72.0),
```

Comment:
722222222222222222222222222222222222222222222222222222222222222222222222222222222222222222

Reply:
64

Reply:
Jak masz to w Row to SizedBox a nie takie paddingi dzikie typu bottom:8, top:16, left: 72, możesz też aligmentem dostosowywać

Reply:
Albo zwrapuj cały Row w padding a nie 50tysięcy razy ten sam padding, albo dobry aligment na Row etc, tak się baw, żeby się nie powtarzać 50 razy z tym samym paddingiem do tego takim ogromnym